Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+name = "mini-starcoin-vm"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+anyhow = "1.0.41"
+
+starcoin-vm-types = {git = "https://github.com/strawWoo/starcoin", rev = "b2bc097f288c63d86d34299a0d6259bb56edf9cb", package="starcoin-vm-types"}
+starcoin-executor = {git = "https://github.com/strawWoo/starcoin", rev = "b2bc097f288c63d86d34299a0d6259bb56edf9cb", package="starcoin-executor"}
+starcoin-state-api = {git = "https://github.com/strawWoo/starcoin", rev = "b2bc097f288c63d86d34299a0d6259bb56edf9cb", package="starcoin-state-api"}
+starcoin-state-store-api = {git = "https://github.com/strawWoo/starcoin", rev = "b2bc097f288c63d86d34299a0d6259bb56edf9cb", package="starcoin-state-store-api"}
+starcoin-statedb = {git = "https://github.com/strawWoo/starcoin", rev = "b2bc097f288c63d86d34299a0d6259bb56edf9cb", package="starcoin-statedb"}
+starcoin-state-tree = {git = "https://github.com/strawWoo/starcoin", rev = "b2bc097f288c63d86d34299a0d6259bb56edf9cb", package="starcoin-state-tree"}
+starcoin-rpc-client = {git = "https://github.com/strawWoo/starcoin", rev = "b2bc097f288c63d86d34299a0d6259bb56edf9cb", package="starcoin-rpc-client"}
```

Comment:
in second run, the crate isn't need.

Reply:
Maybe only the "statcoin-rpc-client" is unnecessary, because others have been used by the process of state-root calculation in "mock_state_node_store.rs".